[2025-04-16T21:29:36.910+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T21:29:36.917+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_dag.read_file manual__2025-04-16T15:59:27.545650+00:00 [queued]>
[2025-04-16T21:29:36.922+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_dag.read_file manual__2025-04-16T15:59:27.545650+00:00 [queued]>
[2025-04-16T21:29:36.922+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-16T21:29:36.932+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): read_file> on 2025-04-16 15:59:27.545650+00:00
[2025-04-16T21:29:36.935+0530] {standard_task_runner.py:72} INFO - Started process 1146 to run task
[2025-04-16T21:29:36.937+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'check_dag', 'read_file', 'manual__2025-04-16T15:59:27.545650+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/check_dag.py', '--cfg-path', '/tmp/tmpgl343i79']
[2025-04-16T21:29:36.938+0530] {standard_task_runner.py:105} INFO - Job 31: Subtask read_file
[2025-04-16T21:29:36.965+0530] {task_command.py:467} INFO - Running <TaskInstance: check_dag.read_file manual__2025-04-16T15:59:27.545650+00:00 [running]> on host Voldemort.
[2025-04-16T21:29:37.015+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='check_dag' AIRFLOW_CTX_TASK_ID='read_file' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T15:59:27.545650+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T15:59:27.545650+00:00'
[2025-04-16T21:29:37.016+0530] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-16T21:29:37.024+0530] {logging_mixin.py:190} INFO - b'Hi there!\n'
[2025-04-16T21:29:37.025+0530] {python.py:240} INFO - Done. Returned value was: None
[2025-04-16T21:29:37.028+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-16T21:29:37.029+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=check_dag, task_id=read_file, run_id=manual__2025-04-16T15:59:27.545650+00:00, execution_date=20250416T155927, start_date=20250416T155936, end_date=20250416T155937
[2025-04-16T21:29:37.070+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T21:29:37.080+0530] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-16T21:29:37.081+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
